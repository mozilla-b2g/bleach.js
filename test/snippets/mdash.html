<p>Closed <a href="https://github.com/mozilla-b2g/B2G/issues/122" class="issue-link" title="Add pkg-config to the mac bootstrap script">#122</a>.</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href='https://github.com/mozilla-b2g/B2G/pull/122'>view it on GitHub</a>.<img src='https://github.com/notifications/beacon/uCMALLF6uny3noj_C7GIZqi74PKRvvJX2aYkvQpkB-AKpuf6lZIjNA4p0Ce9XIZa.gif' height='1' width='1'></p>