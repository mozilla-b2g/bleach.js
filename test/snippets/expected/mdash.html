Closed #122. â€”Reply to this email directly or view it on GitHub.